openslides/core/static/js/core/site.js

```diff
@@ -843,26 +843,57 @@ angular.module('OpenSlidesApp.core.site', [
 // Config Controller
 .controller('ConfigCtrl', [
     '$scope',
+    '$timeout',
     'MajorityMethodChoices',
     'Config',
     'configOptions',
     'gettextCatalog',
     'DateTimePickerTranslation',
     'Editor',
-    function($scope, MajorityMethodChoices, Config, configOptions, gettextCatalog, DateTimePickerTranslation, Editor) {
+    function($scope, $timeout, MajorityMethodChoices, Config, configOptions,
+        gettextCatalog, DateTimePickerTranslation, Editor) {
         Config.bindAll({}, $scope, 'configs');
         $scope.configGroups = configOptions.data.config_groups;
         $scope.dateTimePickerTranslatedButtons = DateTimePickerTranslation.getButtons();
 
         $scope.ckeditorOptions = Editor.getOptions();
         $scope.ckeditorOptions.on.change = function (event) {
-            $scope.save(event.editor.element.$.id, this.getData());
+            // we could just retrieve the key, but we need the configOption object.
+            var configOption_key = event.editor.element.$.id;
+
+            // find configOption object
+            var subgroups = _.flatMap($scope.configGroups, function (group) {
+                return group.subgroups;
+            });
+            var items = _.flatMap(subgroups, function (subgroup) {
+                return subgroup.items;
+            });
+            var configOption = _.find(items, function (_item) {
+                return _item.key === configOption_key;
+            });
+
+            var editor = this;
+            $timeout(function () { // Do this an an angular context
```

Comment:
Done. Yes, this is a normal event. Changes may not directly affect the angular context. See the motion inline factory. The timeouts there do exactly the same.